composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "eb51104c773c3d968b72b17826961fab",
+    "content-hash": "496f231ca5f6ec3243a5b2ccb6b1a2cb",
```

Comment:
I _think_ a `composer update` was required here so `composer.lock` would be updated.